src/odemis/driver/lakeshore.py

```diff
@@ -506,6 +517,7 @@ class SerialBusAccesser(object):
     """
 
     def __init__(self, port: str):
+        # Ensure no one will talk to it simultaneously, and we don't talk to devices already in use
```

Comment:
why is this comment here?

Reply:
incorrectly placed comment(copy-paste error :P), updated it!